dist/routes/posts.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+const express_1 = require("express");
+const posts_1 = require("../controllers/posts");
+const router = express_1.Router();
+// get all posts
+router.get('/', posts_1.getPosts);
+// get one post
+router.get('/:pid', posts_1.getPostById);
+// edit a post -- todo ==> can I just use a post or do I have to use patch??
```

Comment:
The `POST ` Verb is used when creating a new resource. 
When we're talking about updating an _existing_ resource, the two verbs are `PATCH` and `PUT`
`PATCH` is used when you want to modify pieces of the resource
`PUT` is used when you replace the entire resource.
Hope this helps!

Reply:
it totally does!  Thanks!